### PR TITLE
Feature implement filtering

### PR DESCRIPTION
**Pull request by gappc**

### CODE REVIEW

File: databrowser/src/locales/en.json [hunk 1]
```diff
@@ -40,5 +40,11 @@
       "downloadJSON": "Download dataset as JSON",
       "downloadCSV": "Download dataset as CSV"
     }
+  },
+  "components": {
+    "cancelSave": {
```
Comment (alexlanz) [inline databrowser/src/locales/en.json:45]: This seems to me a bit a strange object. Can we remove it and move the two fields just under `components`?

File: databrowser/src/domain/cellComponents/components/filters/CancelSaveButtons.vue [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<template>
```
Comment (alexlanz) [inline databrowser/src/domain/cellComponents/components/filters/CancelSaveButtons.vue:1]: We normally name these buttons the `ControllButtons`. I don't know if you like the name

**Review by alexlanz — APPROVED** [review]
Nice job!

File: databrowser/src/locales/en.json (hunk 1 above)
Comment (gappc) [reply databrowser/src/locales/en.json]: Done with 0713945

File: databrowser/src/domain/cellComponents/components/filters/CancelSaveButtons.vue (hunk 2 above)
Comment (gappc) [reply databrowser/src/domain/cellComponents/components/filters/CancelSaveButtons.vue]: Done with 295d9b6

**Pull request merged**